src/lib/cookies.js

```diff
@@ -107,7 +107,21 @@ class CookieHandler extends CDP {
             await this.Network.deleteCookies(badCookie);
         }
         // Store cookies in the browser
-        await this.Network.setCookies({cookies: browserCookies});
+        const cleanedCookies = browserCookies.reduce((cookies, cookie) => {
+            if (cookie.name !== "") {
+                const cleanedCookie = cookie;
+                if (cleanedCookie.path.includes("%2F")) {
+                    cleanedCookie.path = decodeURIComponent(cleanedCookie.path)
+                } 
+                cookies.push(cleanedCookie);
+            }
+            return cookies;
+        }, [])
+        try {
+            await this.Network.setCookies({cookies: cleanedCookies});
+        } catch (error) {
+            throw new Error (`Could not set cookies. error: ${JSON.stringify(error)} cookies: ${JSON.stringify(cleandCookies)}`)
```

Comment:
JSON.stringify works poorly with error object and their stack traces. Let's just `throw error`

Reply:
what about cookies?